test/integration/db/test_mssql.py

```diff
@@ -119,8 +168,75 @@ def testdb_conn2():
         return conn
 
 
+@pytest.fixture(scope='function')
+def testdb_fast_false_conn():
+    """Get connection to test MS SQL database."""
+    MSSQLDB.fast_executemany = 'false'
+    with connect(MSSQLDB, 'TEST_MSSQL_PASSWORD') as conn:
```

Comment:
Have I missed something?  It seems like we could do:

```python
with connect(MSSQLDB, 'TEST_MSSQL_PASSWORD', use_fast_executemany=True) as conn:
    ...
```

Reply:
You've not missed anything. But in earlier conversations the gist (as I understood it) was to use the parameters, so that's what I have been leaning towards. I may have misunderstood!